src/ngx_http_lua_args.c

```diff
@@ -19,6 +19,56 @@ static int ngx_http_lua_ngx_req_set_uri_args(lua_State *L);
 static int ngx_http_lua_ngx_req_get_post_args(lua_State *L);
 
 
+static ngx_inline ngx_uint_t
+ngx_http_lua_is_escaped_query_string(ngx_http_request_t *r, u_char *str,
+    size_t len)
+{
+    size_t           i, buf_len;
+    u_char           c;
+    u_char          *buf, *src = str;
```

Comment:
Please do not declare multiple variables in a single declarator. We usually don't do this in the nginx code base.